Dont send smartmatrix brightness for old setups

Blackbaud-JohnMarsh opened the pull request:
Don't send the brightness packet for old smartmatrix/teensy setups, if someone is using the old_cookie = true, that means that they are probably using an older setup and the brightness packet will lock up their teensy.

Blackbaud-JohnMarsh committed:
Dont send smartmatrix brightness for old setups

Don't send the brightness packet for old smartmatrix/teensy setups, if someone is using the old_cookie = true, that means that they are probably using an older setup and the brightness packet will lock up their teensy.

jabdoa2 commented:
This test fails:

```

======================================================================
FAIL: test_smart_matrix (test_SmartMatrix.TestSmartMatrix)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/missionpinball/mpf/mpf/tests/test_SmartMatrix.py", line 70, in test_smart_matrix
    call(b'\x01\x00\x01\x02\x03')                               # frame
  File "/usr/lib/python3.5/unittest/mock.py", line 824, in assert_has_calls
    ) from cause
AssertionError: Calls not found.
Expected: [call(b'\xba\x11\x00\x03\x14\x7f\x00\x00'), call(b'\x01\x00\x01\x02\x03')]
Actual: [call(bytearray(b'\x01\x00\x01\x02\x03'))]

```

Blackbaud-JohnMarsh committed:
Remove brightness packet from oldcookie smartmatrx

Remove the brightness packet from the test for oldcookie smartmatrix interfaces.

Blackbaud-JohnMarsh commented:
I've removed the brightness packet from the old cookie test

jabdoa2 commented:
Thanks John!
